<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visiteur</class>
 <widget class="QDialog" name="visiteur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Identifiant :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nom :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Prénom :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Adresse :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Code postal :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Ville :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Secteur :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Laboratoire :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Département :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Hiérarchie :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Chef de vente :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditMatricule"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditNom"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPrenom"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAdresse"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCP">
         <property name="maxLength">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditVille"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxSecteur"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxLaboratoire"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxDepartement"/>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButtonDelegue">
            <property name="text">
             <string>Délégue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonResponsable">
            <property name="text">
             <string>Responsable</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonVisiteur">
            <property name="text">
             <string>Visiteur</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxChefVente">
         <property name="text">
          <string>Chef de vente du département</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonAjouter">
       <property name="text">
        <string>Editer</string>
       </property>
       <property name="icon">
        <iconset resource="Images.qrc">
         <normaloff>:/enregistrer</normaloff>:/enregistrer</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="Images.qrc">
         <normaloff>:/quitter</normaloff>:/quitter</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
